api-client/package.json

```diff
@@ -20,7 +20,7 @@
     "@babel/runtime": "7.13.7",
     "js-md5": "^0.7.3",
     "entities": "^2.0.0",
-    "moment": "2.25.3",
+    "moment": "2.29.1",
```

Comment:
:fearful: Hopefully this does not break anything :D

Reply:
So I checked the release notes and it seems, that there were no breaking changes. However I found [this](https://momentjs.com/docs/#/-project-status/) interesting article about moment becoming a legacy project o.o

Reply:
Yeah crazy, right? We already have an issue to switch away from moment but we wanted to wait until we get correct timestamps from the cms which will only be the case with django.